sdk/inc/azure/iot/az_iot_adu_client.h

```diff
@@ -0,0 +1,532 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
```

Comment:
```suggestion
 * @file az_iot_adu_client.h
```

Reply:
learned from victor that this isn't actually necessary (and is debatably better without it since there's isn't an error if you change the file name).